spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/authentication/securityPolicy/SecurityPolicyAccessController.java

```diff
@@ -38,17 +44,41 @@ public SecurityPolicyAccessController(AuthenticationAdapter authenticationAdapte
   }
 
   @Override
-  public boolean isAllowed(String serviceId, String instanceId, Map<String, String> requestMap) {
+  public void valid(String token, Map<String, String> requestMap) throws Exception {
+    RsaAuthenticationToken rsaToken = checkTokenInfoOrServiceName(token);
```

Comment:
when token check is enabled while is not valid, should fail.

Reply:
fixed